### PR TITLE
feat(other): add custom CSS for Authentik flows (desktop)

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
Add 

### Sign In Flow
custom_css_sign_in

### Sign Up Flow
custom_css_sign_up

### Issues
- relates #375

### Todo
- [X] resize logo to 60% width
- [ ] input fields
  - [x] `focus`: linear-radient outline
  - [ ] `invalid`: red border
  - [x] else: black border
  - [ ] resize
- [ ] buttons
  - [x] final buttons (login, registration close): green
  - [ ] continue buttons:  blue - 
  - [ ] capitalize text
  - [ ] resize
- [ ] footer links
  - [x] add specified links (even without proper target yet)
  - [ ] remove default links

### CODE REVIEW

**Review by ulfgebhardt — COMMENTED** [review]
Is this on production? I cannot see it?

Can you provide screenshots?

**Comment by mahula** [comment]
> Is this on production? I cannot see it?

Since this PR has not been approved yet, thes changes are not deployed anywhere.

> Can you provide screenshots?

Sure. See the PR description.
This PR can be tested locally.

**Comment by mahula** [comment]
@ulfgebhardt 
now the changed database is in the PR.
Sorry, my bad.

File: authentik/docker-compose.yml [hunk 1]
```diff
@@ -44,6 +44,7 @@ services:
       - ./media:/media
       - ./custom-templates:/templates
       - ./data:/data
+      - ./media/custom.css:/web/dist/custom.css
```
Comment (Elweyn) [inline authentik/docker-compose.yml:47]: Would be better to have this:
```suggestion
      - ./web/dist/custom.css:/web/dist/custom.css
```

File: authentik/media/custom.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/* brand logo */
```
Comment (Elweyn) [inline authentik/media/custom.css:1]: This file should lay under authentik/web/dist/

File: authentik/docker-compose.yml (hunk 1 above)
Comment (mahula) [reply authentik/docker-compose.yml:47]: `./web/dist` does not exist in the repo, it wil be created in a build process.
So what about placing the custom CSS file in `./custom.css` and binddit in the YAML file this way `./custom.css:/web/dist/custom.css`?

**Review by Tirokk — APPROVED** [review]
Cool things 😍

I have tested it and approve 🚀🚀💫

**Pull request merged**